dataactcore/utils/jobQueue.py

```diff
@@ -1,36 +1,90 @@
 from celery import Celery
-from dataactcore.config import CONFIG_DB, CONFIG_SERVICES, CONFIG_JOB_QUEUE
+from dataactcore.config import CONFIG_DB, CONFIG_SERVICES, CONFIG_JOB_QUEUE, CONFIG_BROKER
 import requests
+from dataactvalidator.filestreaming.csvS3Writer import CsvS3Writer
+from csv import reader
+from dataactcore.aws.s3UrlHandler import s3UrlHandler
 
 class JobQueue:
+    def __init__(self, job_queue_url="localhost"):
+        # Set up backend persistent URL
+        backendUrl = ''.join(['db+', CONFIG_DB['scheme'], '://', CONFIG_DB['username'], ':', CONFIG_DB['password'], '@',
+                              CONFIG_DB['host'], '/', CONFIG_DB['job_queue_db_name']])
 
-	def __init__(self):
-		# Set up backend persistent URL
-		backendUrl = ''.join(['db+', CONFIG_DB['scheme'], '://', CONFIG_DB['username'], ':', CONFIG_DB['password'], '@', CONFIG_DB['host'], '/', CONFIG_DB['job_queue_db_name']])
+        # Set up url to the validator for the RESTFul calls
+        validatorUrl = ''.join([CONFIG_SERVICES['validator_host'], ':', str(CONFIG_SERVICES['validator_port'])])
+        if 'http://' not in validatorUrl:
+            validatorUrl = ''.join(['http://', validatorUrl])
 
-		# Set up url to the validator for the RESTFul calls
-		validatorUrl = ''.join([CONFIG_SERVICES['validator_host'], ':', str(CONFIG_SERVICES['validator_port'])])
-		if 'http://' not in validatorUrl:
-			validatorUrl = ''.join(['http://', validatorUrl])
+        # Set up url to the job queue to establish connection
+        queueUrl = ''.join(
+            [CONFIG_JOB_QUEUE['broker_scheme'], '://', CONFIG_JOB_QUEUE['username'], ':', CONFIG_JOB_QUEUE['password'],
+             '@', job_queue_url, ':', str(CONFIG_JOB_QUEUE['port']), '//'])
 
-		# Set up url to the job queue to establish connection
-		queueUrl = ''.join([CONFIG_JOB_QUEUE['broker_scheme'], '://', CONFIG_JOB_QUEUE['username'], ':', CONFIG_JOB_QUEUE['password'], '@', CONFIG_JOB_QUEUE['url'], ':', str(CONFIG_JOB_QUEUE['port']), '//'])
+        # Create remote connection to the job queue
+        self.jobQueue = Celery('tasks', backend=backendUrl, broker=queueUrl)
 
-		# Create remote connection to the job queue
-		self.jobQueue = Celery('tasks', backend=backendUrl, broker=queueUrl)
+        @self.jobQueue.task(name='jobQueue.enqueue')
+        def enqueue(jobID):
+            # Don't need to worry about the response currently
+            url = ''.join([validatorUrl, '/validate/'])
```

Comment:
When we originally put this in there, we had just set it up to call /validate. I noticed validate_threaded but it wasn't ever used so I didn't bother messing with it. We could utilize it, maybe that would help with the validations being blocked